package/functions/funcs/onlyForRoles.js

```diff
@@ -8,13 +8,13 @@ module.exports = async d => {
 	
 	if (err) return d.error(err)
 
-    const IDs = inside.splits
+    let fields = inside.splits
 
-    const errorMessage = IDs.pop()
-``
+    const errorMessage = fields.pop()
+const IDs = fields
```

Comment:
IDs.pop() is changing the original array, assigning to new variable might be a redundant copy